### DISCUSSION TITLE
Date format

### DISCUSSION THREAD

**Discussion by Cronycs** [protz/thunderbird-conversations#1676] [Q&A] [answered]
Hello,
I am using "Conversation" since many years now and I found it very useful, thank you for this great plugin :)
I am using the "Nice date" feature but since few weeks, it is not showing anymore the hour:minute in addition to the date. Is it because of a Thunderbird update or an update of the plugin?

**Comment by Standard8** [comment, marked as answer]
Yeah, so this is currently broken in when using Thunderbird 78, but works in Thunderbird 91 which should be coming out soon.

The issue is that TB 78 doesn't have support for a newer API that I needed to use, and working around that is hard. As I'm not seeing significant numbers of people complaining, I don't really want to revert / implement a work around at this stage, as I'm mainly concentrating on fixing lots of broken things with TB 91.

**Reply by Cronycs** [reply]
Thanks for the reply !
I'll patiently wait for Thunderbird 91 :-)
